calculator.js

```diff
@@ -1,5 +1,114 @@
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
+// This line "loads" the prompt package and puts it into a variable we can use called prompt
+const prompt = require('prompt');
+
+console.log('Welcome to the Calculator program! Please provide 2 integers or floats and then choose an operator');
+console.log('1. add (+)');
+console.log('2. subtract (-)');
+console.log('3. multiply (*)');
+console.log('4. divide (/)');
+console.log('5. exponify (^)');
+console.log('6. find a remainder (%)');
+
+const calculateUserInput = function(error, promptInput) {
+  console.log(operation(Number(promptInput.num1), Number(promptInput.num2), promptInput.operator.toLowerCase()))
+}  
```

Comment:
This is not very intuitive but this is still a variable definition so a `;` technically goes at the end.
JS is super flexible so it works hard to add them for you when it's pretty sure you forgot one, but our best practice at Ada is to try to add them everywhere they go rather than relying on JS to do that for us.